src/Controller/UserController.php

```diff
@@ -89,16 +90,16 @@ public function editUser($id, SerializerInterface $serializer, Request $request,
         $data = $request->getContent();
```

Comment:
We forgot to add a check that you can edit only your profile. Please compare the logged in user with the user which is being edited.